assignments/character_count.md

```diff
@@ -0,0 +1,18 @@
+##Character Count
+
+Given an array of colors, return a hash for each color with each character's character count. E.g.,
+colors = ["purple", "yellow", "green", "red"]
+
+green_hash = { "g": 1, "r": 1, "e": 2, "n": 1}
+red_hash = { "r": 1, "e": 1, "d": 1}
+purple_hash = { "p": 2, "e": 1, "u": 1, "l": 1, "r": 1}
+
+## Notes
+
+Remember, order doesn't matter in hashes.
+
+## Additional Challenges
+
+Have your program accept an argument of an array
+Return the character with the highest count for each item
```

Comment:
Potentially a good springboard into objects.

Reply:
yeah, I definitely saw some openings for teaching OOP in this lesson, though I think will wait to go through the on_bill problem a little more before introducing.